Source/ARTRealtime.m

```diff
@@ -1162,6 +1180,7 @@ - (void)sendImpl:(ARTProtocolMessage *)pm sentCallback:(ARTCallback)sentCallback
     
     if (pm.ackRequired) {
         self.msgSerial++;
+        self.connection.latestMessageSerial = self.msgSerial;
```

Comment:
I'm slightly uncomfortable, from a readability perspective, that we have the word "message" in raw and abbreviated forms in the names of two properties which represent the same thing. Might we consider replacing occurrences of `messageSerial` in this codebase with `msgSerial` to better match with the specification? (I usually advocate for avoiding abbreviations, however I recognise that the `msg` abbreviation is baked into our spec.).